internal/exec/terraform_outputs.go

```diff
@@ -296,19 +296,15 @@ func GetTerraformOutput(
 
 	sections, err := ExecuteDescribeComponent(component, stack, true, true, nil)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
+		l.Debug(fmt.Sprintf("\r✗ %s\n", message))
 		l.Fatal("Failed to describe the component", "component", component, "stack", stack, "error", err)
 	}
 
 	// Check if the component in the stack is configured with the 'static' remote state backend, in which case get the
 	// `output` from the static remote state instead of executing `terraform output`
 	remoteStateBackendStaticTypeOutputs, err := GetComponentRemoteStateBackendStaticType(sections)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
+		l.Debug(fmt.Sprintf("\r✗ %s\n", message))
```

Comment:
I think the original implementation was wrong to use log levels.

This is console output, versus log output. 

We should probably have a function dedicated to console output, so it's a deliberate thing, versus the fmt.Printf.

Log Output: Primarily meant for developers, operations teams, and debugging. It captures events, errors, and system state based on log levels (DEBUG, INFO, WARN, ERROR, etc.).

Console Output: Primarily meant for users, especially in interactive or CLI applications. It presents structured UI elements, progress indicators, tables, or other human-readable content.

The "x" is the giveaway this console output and not log output. Console output is not governed by log levels.

Reply:
Thanks for the clarification

Reply:
Done